base/applications/mspaint/registry.cpp

```diff
@@ -61,6 +61,23 @@ void RegistrySettings::LoadPresets()
     ThumbXPos = 180;
     ThumbYPos = 200;
     UnitSetting = 0;
+    Bold = FALSE;
+    Italic = FALSE;
+    Underline = FALSE;
+    CharSet = DEFAULT_CHARSET;
+    PointSize = 14;
+    FontsPositionX = 0;
+    FontsPositionY = 0;
+    ShowTextTool = TRUE;
+
+    {
+        LOGFONT lf;
+        GetObject(GetStockObject(DEFAULT_GUI_FONT), sizeof(lf), &lf);
+        strFontName = lf.lfFaceName;
+    }
```

Comment:
In C++ it's fine, you don't need to use a new code block scope to define your new variable in the middle.
```suggestion
    LOGFONT lf;
    GetObject(GetStockObject(DEFAULT_GUI_FONT), sizeof(lf), &lf);
    strFontName = lf.lfFaceName;
```

Reply:
OK.